bool ask_tag
---
geometry_msgs/Point tag_result
